### ISSUE TITLE
bug: the button that expands the followers list found at the top is not labeled according to my screen reader.

### ISSUE THREAD

**Issue by marrie** [lbryio/lbry-desktop#5185]
**Bug**

When tabbing or navigating wiht the arrow keys, you will hear "button" at the very top of the screen, if using a screen reader. when pressing this button, the list of who you are following will appear. The button has no label, thereby making it harder for a screen reader user to find the button and go to their follower list.

**To Reproduce**

I'm using nvda which can be found [ere.](http://nvda-project.org)

1. launch nvda.
2. launch the lbry app or the odysee website.
3. turn off your screen.
4. use the keyboard to navigate to the "button" It will say "button."
5. hit enter on the button.
6. tab or use arrows to find your followers list.

**Expected behavior**

I should hear what that button does. "see followers" o or what ever that label actually is supposed to be.

**Screenshots**

**System Configuration**

![screenshot of the about section in the help part of the lbry app.](https://user-images.githubusercontent.com/3614487/102175475-7c637580-3e54-11eb-862d-494ec5db4fdb.png)

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Additional context**
n/a

**Comment by seanyesmunt** [comment]
Thanks for reporting this @marrie 

I will make sure this is fixed soon

**Comment by marrie** [comment]
I'm still able to reproduce this under mac and windows. Thanks so much and happy Sunday.

**Comment by marrie** [comment]
Thanks for the fix. That was fast.￼

**Comment by seanyesmunt** [comment]
Thanks for closing the issue! I forgot about that. 🙂 

If you run into any more screen reader issues please create another issue so we can fix it.
